sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -58,11 +57,9 @@ public override void Process(HttpMessage message, ReadOnlyMemory<HttpPipelinePol
 
         private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline, bool async)
         {
-#if NETCOREAPP2_1
-            using var scope = new DiagnosticScope("Azure.Core.Http.Request", s_diagnosticSource, message, s_activitySource, DiagnosticScope.ActivityKind.Client, false);
+#if NETCOREAPP2_1 // Tracing is disabled in netcoreapp2.1
```

Comment:
this one should stay, but as 
```suggestion
#if NETCOREAPP2_1 // Tracing is disabled in netcoreapp2.1
using var scope = new DiagnosticScope("Azure.Core.Http.Request", s_diagnosticSource, message, null, System.Diagnostics.ActivityKind.Client, false);
```